CONTRIBUTING.md

```diff
@@ -25,13 +25,22 @@ aiconfig/
 ├─ editor/
 ├─ aiconfig-docs/
 ├─ python/
-│   ├── model-parsers/
 ├─ typescript/
-│   ├── model-parsers/
 ├─ extensions/
 ```
 
-> *To Do: Add information on library structure and contribution opportunities*
+1. **{ Model Parsers }**: Model Parsers manage the conection to foundation models. By default, we provide the OpenAI models and Google PaLM models. The model parsers are defined in both `python` and `typescript`. 
```

Comment:
nit, `conection` -> `connection`
`we provide the OpenAI ...` --> `we provide model parsers for OpenAI ...`